crates/bevy_app/src/app.rs

```diff
@@ -328,7 +328,7 @@ impl App {
 
     /// Adds [`State<S>`] and [`NextState<S>`] resources, [`OnEnter`] and [`OnExit`] schedules
     /// for each state variant (if they don't already exist), an instance of [`apply_state_transition::<S>`] in
-    /// [`StateTransition`] so that transitions happen before [`Update`] and
+    /// [`StateTransition`] so that transitions happen before [`crate::Update`] and
```

Comment:
I think that displays "crate::Update" with the link, contrary to [`Update`](crate::Update) ? (I didn't check)